alerter/build.gradle

```diff
@@ -19,7 +19,7 @@ final String GITHUB_URL   = "https://github.com/Tapadoo/Alerter"
 
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
-    buildToolsVersion rootProject.ext.buildToolsVersion
+    buildToolsVersion '28.0.2'
```

Comment:
I'd recommend updating 'rootProject.ext.buildToolsVersion' instead, as it is used in multiple places in the project.

Reply:
will do, thanks.

Reply:
oops, will do once I get off work. Thanks!